src/main/java/org/broadinstitute/consent/http/db/mapper/DarCollectionSummaryReducer.java

```diff
@@ -25,12 +25,19 @@ public void accumulate(Map<Integer, DarCollectionSummary> map, RowView rowView)
     Integer datasetId;
     String darStatus;
     String darReferenceId;
+
     try {
       datasetId = rowView.getColumn("dd_datasetid", Integer.class);
       if (Objects.nonNull(datasetId)) {
         summary.addDatasetId(datasetId);
       }
 
+      if (hasColumn(rowView, "dac_name", String.class)) {
+        if (Objects.nonNull(rowView.getColumn("dac_name", String.class))) {
+          summary.addDacName(rowView.getColumn("dac_name", String.class));
+        }
+      }
+
       try {
         darReferenceId = rowView.getColumn("dar_reference_id", String.class);
```

Comment:
can we add a test to make sure the DAO calls properly populate this? otherwise LGTM